projects/managed-hubs/support/process.md

```diff
@@ -153,6 +154,30 @@ When a new ticket lands in Freshdesk under the support group and it is not an in
 
     **Close the request**. If the Community Representative confirms that their request has been fulfilled, consider this request closed. In FreshDesk, mark the incident as {guilabel}`Closed`.
 
+(support:end-of-rotation-hand-off)=
+### End of rotation hand-off
+
+Below is the process describing the expectations before a triager's end of rotation.
+
+1. Processed ~all unassigned tickets
+
+_Support triagers should aim to process each unassigned ticket before the end of the rotation._
+
+This helps us ensure no tickets becomes stale and unattended, and nudges us to get tickets closed or resolved rather than stuck open or pending indefinitively. Tickets left open becomes weight over time.
+
+2. Unassign from tickets
+
+_Support triagers should by end of rotation unassign themselves as agents from all tickets._
+
+This helps making responsibilities unambigious. A ticket's assigned agent should be either nobody or one of the current triagers.
+
+3. Write handover notes
+
+_Support triagers should, when they unassign from a ticket ensure a handover note is available and up to date towards the end of the ticket._
+
+A ticket handover note should _summarize whats relevant_ to know in order to drive the ticket towards a resolution without needing to read through the conversation up to that point.
```

Comment:
```suggestion
A ticket handover note should _summarize whats relevant_ to know in order to drive the ticket towards a resolution without needing to read through the conversation up to that point.

This should be written using the [private notes](https://support.freshdesk.com/en/support/solutions/articles/37580-private-notes-for-internal-sharing)
feature of freshdesk.
```

This should be written as a Freshdesk note.

Reply:
Excellent yes - thank you!